### PR TITLE
Improve Gatecrasher's narrative, enemies, and hack rewards

### LINKED ISSUE #534: Gatecrasher needs improvements
https://discordapp.com/channels/578606792448409610/656035944058847232/720573300748583032

### PR DESCRIPTION
**Pull request by cannonfodder1**
Closes #534 

Hidden within the kismet file, I used the new SeqAct I made to change the doors' hack defense from 60 to 10. With the standard soldier hack offense of 5, that means the chance to get the T1 hack reward is around 35%, which should incentivize freeing the prisoners first rather than clearing the site before heading to the cells. All other changes are visible in the github diffs.

### CODE REVIEW

File: CovertInfiltration/Localization/XComGame.int [hunk 1]
```diff
@@ -351,18 +351,14 @@ ConsoleObjectiveTextPools[0]=
 ConsoleObjectiveTextPools[1]=
 
 [GatecrasherCI X2MissionTemplate]
-DisplayName="Operation Gatecrasher"
-Briefing="ADVENT has brought a pair of dissident academics to this facility for 're-education'. These two could be of extreme value to the Resistance. Ensure they both make it out of here alive."
+DisplayName="Rescue VIPs from ADVENT Compound"
```
Comment (Xymanek) [inline CovertInfiltration/Localization/XComGame.int:354]: I think you should keep the `DisplayName="Operation Gatecrasher"` part

File: CovertInfiltration/Localization/XComGame.int [hunk 2]
```diff
@@ -351,18 +351,14 @@ ConsoleObjectiveTextPools[0]=
...
+Briefing="Rescue VIPs from ADVENT Compound"
```
Comment (Xymanek) [inline CovertInfiltration/Localization/XComGame.int:355]: In line with "Gatecrasher", could this be something like "break into advent compound and rescue VIPs"?

File: CovertInfiltration/Localization/XComGame.int (hunk 1 above)
Comment (cannonfodder1) [reply CovertInfiltration/Localization/XComGame.int:354]: All the other mission locs make both DisplayName and Briefing the exact same thing

Comment (Xymanek) [reply CovertInfiltration/Localization/XComGame.int:354]: Uhhh... not all

```ini
[CovertEscape X2MissionTemplate]
DisplayName="Escape Covert Action"
Briefing="Covert Action - Escape ADVENT Counterops Team"

[SupplyExtraction X2MissionTemplate]
DisplayName="Extract ADVENT Supplies"
Briefing="Supply Raid - Procure ADVENT Supplies"

[NeutralizeFieldCommander X2MissionTemplate]
DisplayName="Neutralize Field Commander"
Briefing="Neutralize ADVENT Field Commander"
```

**Comment by Xymanek** [comment]
Did you fix the mission fail screen?

**Comment by Xymanek** [comment]
Also, did you check what's going on with the VIPs' "culture"?

**Comment by cannonfodder1** [comment]
> Did you fix the mission fail screen?

Fix what? I thought it was fine?

> Also, did you check what's going on with the VIPs' "culture"?

I did some tests, the names matched the region. I feed the region state into the generate reward, so it should be working fine.

**Comment by Xymanek** [comment]
> Fix what? I thought it was fine?

When I tried it, I got "mission failed, continue" screen, while it needs to be "unfailable mission failed, retry"? 

> I did some tests, the names matched the region

Ok then, must've been something weird on my end

**Pull request merged**